pkg/apis/jvmbuildservice/v1alpha1/dependencybuild_types.go

```diff
@@ -74,4 +74,6 @@ type BuildRecipe struct {
 	CommandLine      []string `json:"commandLine,omitempty"`
 	EnforceVersion   string   `json:"enforceVersion,omitempty"`
 	IgnoredArtifacts []string `json:"ignoredArtifacts,omitempty"`
+	ToolVersion      string   `json:"toolVersion,omitempty"`
+	JavaHome         string   `json:"javaHome,omitempty"`
```

Comment:
This should be JavaVersion.

Reply:
Done. Switched from `JavaHome` to `JavaVersion`. It will still set `JAVA_HOME` from the yaml using that version.